### DISCUSSION TITLE
Is parmas participation better for seo than query participation for router?

### DISCUSSION THREAD

**Discussion by jameszhanc** [nuxt/nuxt#24865] [Questions]
Is parmas participation better for seo than query participation for router?

**Comment by TheAlexLichter** [comment]
I suppose you ask: "Are route parameters better than query parameters for SEO?"

Then I'd say: Not *necessarily*, but it is often better:

* Cleaner URL structure
* No sorting issue (?a=1&b=2 vs. ?b=2&a=1)
* No duplicate content (decide which query parameters are *mandatory* to show the content and not superfluous)
